### PR TITLE
pass cancellation

### PR DESCRIPTION
**Pull request by cameronpettit**
Note: Email service currently does not check front-end env before generating cancellation links, therefore the generated links point to the wrong API. Front-End should still read url query params properly.

### CODE REVIEW

File: src/app/shared/components/alert/alert-object.ts [hunk 1]
```diff
@@ -20,6 +20,11 @@ export interface IAlertObjectParams {
    */
   type?: string;
 
+  /**
+   * size of Alert - if true, Alert is single line. If false, alert is multiple lines
+   */
+  smallAlert?: boolean;
```
Comment (MaxWardle) [inline src/app/shared/components/alert/alert-object.ts:26]: do we still use alerts or is it exclusively toast now? I don't remember seeing any in testing

**Review by MaxWardle — COMMENTED** [review]
Looks good in testing on mobile, and works functionally. Just one comment about the alerts, if they're not used we should probably get rid of them. Nice work though 👏

File: src/app/shared/components/alert/alert-object.ts (hunk 1 above)
Comment (cameronpettit) [reply src/app/shared/components/alert/alert-object.ts]: removing alerts, toasts indeed way better

**Review by mtCarto — APPROVED** [review]

**Review by MaxWardle — APPROVED** [review]
looks good :)

**Pull request merged**